internal/display/apis.go

```diff
@@ -68,7 +68,7 @@ func (v *apiTableView) Object() interface{} {
 }
 
 func (r *Renderer) ApiList(apis []*management.ResourceServer) {
-	resource := "APIs"
```

Comment:
We are also making all the headings that appear after the tenant's name lowercase for consistency.